components/mixins/profile/profile-user-account-details.schema.json

```diff
@@ -61,13 +61,13 @@
               "type": "string",
               "description": "Captures the user account type, if applicable."
             },
-            "xdm:preferences": {
-              "title": "Preferences",
+            "xdm:shoppingPreferences": {
+              "title": "Shopping Preferences",
```

Comment:
Is there a way we can make this more generic without having shopping in front? I want tit to apply for travel/hosp also. So bed prefs and seating prefs. Would product preferences be more generic?

Reply:
categoryPreferences?